lib/oneview-sdk/resource/api300/synergy/logical_interconnect.rb

```diff
@@ -9,12 +9,33 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-require_relative '../../api200/logical_interconnect'
+require_relative '../c7000/logical_interconnect'
 
 module OneviewSDK
   module API300
     module Synergy
-      class LogicalInterconnect < OneviewSDK::API200::LogicalInterconnect
+      # Logical interconnect resource implementation for API300 Synergy
+      class LogicalInterconnect < OneviewSDK::API300::C7000::LogicalInterconnect
+
+        # Lists internal networks on the logical interconnect
+        # @return [OneviewSDK::Resource] List of networks
+        def list_vlan_networks
```

Comment:
That's fine. I'm hitting a few failing tests on my appliances, but it might be my conf. If all integrations for both C7000 and Synergy are passing there, just fix the ``CHANGELOG.md`` and it should be good to merge.